src/discord/commands/pdfs.ts

```diff
@@ -11,11 +11,13 @@ export default {
         .setRequired(true)
     )
     .setDescription(
-      'Skicka in en årsredovisning och få tillbaka utsläppsdata.'
+      'Skicka in flera årsredovisningar och få tillbaka utsläppsdata.'
```

Comment:
If we remove the `/pdf` command, this should probably say `en eller flera årsredovisningar`

Reply:
True. Since it's a separate PR that hasn't been merged yet, I think that would be a better place for the change though.